### PR TITLE
CSS => SCSS, Page admin feature fixes

### PR DESCRIPTION
**Pull request by denny**
Move admin area to SCSS for great justice / colour name variables, and fix the drop-downs in the admin edit pages to reselect the current setting. And stuff.

### CODE REVIEW

File: app/views/admin/pages/edit.html.erb [hunk 1]
```diff
@@ -19,13 +19,13 @@
   </p>
   <p>
     Template:
-    <br><%= select( 'page', 'template_id',
-      PageTemplate.all.collect { |t| [ t.name, t.id ] } ) %>
+    <br><%= select( 'page', 'template_id', PageTemplate.pluck( :name, :id ),
```
Comment (eliotsykes) [inline app/views/admin/pages/edit.html.erb]: Consider using the form builder (`f`) `select` method, and it ought to set the `selected` option correctly:

```erb
<%= f.select(:template_id,  PageTemplate.pluck( :name, :id ) %>
```

File: app/views/admin/pages/edit.html.erb [hunk 2]
```diff
@@ -19,13 +19,13 @@
...
+            { selected: @page.template_id} ) %>
   </p>
   <p>
     Section:
-    <br><%= select( 'page', 'section_id',
-      PageSection.all.collect { |s| [ s.name, s.id ] }, { include_blank: 'None' } ) %>
+    <br><%= select( 'page', 'section_id', PageSection.pluck( :name, :id ),
```
Comment (eliotsykes) [inline app/views/admin/pages/edit.html.erb]: As above, try form builder `select` method: `f.select(...)`

Comment (eliotsykes) [reply app/views/admin/pages/edit.html.erb]: Also consider using `f.label(:section_id, "Section")` for a11y (line 26).

File: app/views/admin/pages/index.html.erb [hunk 3]
```diff
@@ -36,13 +33,63 @@
...
       <%= link_to 'View', url_for( "/pages/#{section.slug}/#{page.slug}" ) %>
-      <%= link_to 'Edit', url_for( controller: '/admin/pages', action: 'edit', id: page.id ) %>
+      <%= link_to 'Edit', url_for( controller: '/admin/pages', action: 'edit', id: page ) %>
 		</td>
 	</tr>
-	<% end %>
-	<% end %>
+
+  <% indent = indent + 2 %>
+  <% section.sections.each do |subsection| %>
```
Comment (eliotsykes) [inline app/views/admin/pages/index.html.erb]: The table row/cell generation code looks like a good candidate for DRYing up with partial template(s).

File: app/views/admin/pages/edit.html.erb (hunk 1 above)
Comment (denny) [reply app/views/admin/pages/edit.html.erb]: Aha, cool, _much_ neater.

What you see here is what I found when googling for syntax, I did wonder why it wasn't an f.* like the rest.

File: app/views/admin/pages/index.html.erb (hunk 3 above)
Comment (denny) [reply app/views/admin/pages/index.html.erb]: Yeah, particularly as it's a recursive data structure ("It's turtles all the way down!")

Comment (denny) [reply app/views/admin/pages/index.html.erb]: Boom! https://github.com/denny/ShinyCMS-ruby/pull/35/commits/5aaf4715161b8aced8c1d2f93956543c2b8560c8

**Pull request merged**